examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -63,8 +63,8 @@ grid = RectilinearGrid(CPU();
 
 fig = Figure(resolution=(1200, 800))
 ax = Axis(fig[1, 1], ylabel = "Depth (m)", xlabel = "Vertical spacing (m)")
-lines!(ax, grid.Δzᵃᵃᶜ[1:grid.Nz], grid.zᵃᵃᶜ[1:grid.Nz])
-scatter!(ax, grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz])
+lines!(ax, zspacings(grid, Center())[1:grid.Nz], znodes(grid, Center())[1:grid.Nz])
+scatter!(ax, zspacings(grid, Center())[1:Nz], znodes(grid, Center())[1:Nz])
```

Comment:
Why do we need the `1:Nz`?

Reply:
to avoid showing the spacings for halo points... but can't we do this somehow else?

Reply:
Like with an optional flag for `zspacing()`?

Reply:
Should we simply define these functions so that they don't include halo points?

Reply:
I think an optional flag would be OK. These are not function that are used in time-stepping so it should be alright (perfomance-wise). E.g., `zspacings(grid, Center(); include_halos = true)` if we want to include the halos; otherwise `include_halos = false` by default?

`interior(zspacings(grid, Center()))` would also be intuitive but this requires that `zspacings` returns a field... and we probably don't wanna go down that road.

Reply:
I think we should return fields eventually but only once we have plotting recipes for Makie.